src/.editorconfig

```diff
@@ -0,0 +1,4 @@
+root = true
+
+[*]
+end_of_line = lf
```

Comment:
OK, missed the root .editorconfig - much better than mine, I will remove mine.

This is really a minor annoyance for me, the rest of the comment goes into the details.

The reason I suggested adding end_of_line was because I got a "mixed line ending" warning from Visual Studio where my contributions had CRLF whereas the rest of the file was LF.
Shouldn't it be project specific rather than developer?

However my initial problem turned out to be caused by a bug in the Visual studio /// snippet that always have line endings using CRLF, the .editorconfig doesn't fix this at this moment. It is triaged in the visual studio feedback system.

For every new file I make a change in I get the "mixed line ending" when I open the file the next time. If I'm not careful, and only choose ok, it will convert the rest of the file to CRLF endings.
As a lazy Visual Studio user I would suggest change the line endings to CRLF mainly for the reason to not get annoyed by the ever appearing "mixed line ending" warnings.

Reply:
Normally, when working on Windows, if you are using the default installation settings for Git for Windows, it should automatically convert the line endings of text files to `CRLF` upon pulling and back to `LF` when pushing to remote. The setting is called `core.autocrlf` IIRC.